cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/CaReconciler.java

```diff
@@ -109,7 +114,8 @@ public CaReconciler(
             ResourceOperatorSupplier supplier,
             Vertx vertx,
             CertManager certManager,
-            PasswordGenerator passwordGenerator
+            PasswordGenerator passwordGenerator,
+            KubernetesRestartEventPublisher eventPublisher
```

Comment:
Why isn't it taken from the ResourceOperatorSupplier?

Reply:
:+1: